src/Elmish.WPF/ViewModelUtilities.fs

```diff
@@ -0,0 +1,28 @@
+﻿[<RequireQualifiedAccess>]
+module Elmish.WPF.Utilities.ViewModel
+
+open System.Windows
+open Elmish
+open Elmish.WPF
+open Elmish.WPF.Internal
+
+/// Start Elmish dispatch loop
+let internal startLoop
+    (config: ElmConfig)
+    (element: FrameworkElement)
+    (programRun: Program<'t, 'model, 'msg, BindingSpec<'model, 'msg> list> -> unit)
+    (program: Program<'t, 'model, 'msg, BindingSpec<'model, 'msg> list>) =
+  let mutable lastModel = None
+
+  let setState model dispatch =
+    match lastModel with
+    | None ->
+        let mapping = program.view model dispatch
+        let vm = ViewModel<'model,'msg>(model, dispatch, mapping, config, (fun f -> element.Dispatcher.Invoke f))
```

Comment:
This line is a bit long now. Could you define `let uiDispatch (f: unit -> unit) = element.Dispatcher.Invoke f` and use this in the constructor? I think that's the last change.